docs/pages/_app.js

```diff
@@ -275,6 +280,20 @@ function findActivePage(currentPages, pathname) {
   return activePage;
 }
 
+// Cache for the ltr version of the styles
+const cacheLtr = createCache({
+  key: 'mui',
+  stylisPlugins: [],
+  speedy: true,
```

Comment:
What is speedy for? It's not documented https://emotion.sh/docs/@emotion/cache. Could it be an internal https://github.com/emotion-js/emotion/blob/c85378a204613885a356eaba1480c5151838c458/packages/cache/src/index.js#L243?

Reply:
I saw it used internally on their side, but you are correct, it seems to be internal. Will remove it for now..